pillar/heroku/ocw-studio.sls

```diff
@@ -190,6 +190,7 @@ heroku:
     THREEPLAY_API_KEY: __vault__::secret-operations/global/{{ business_unit }}/ocw-studio/threeplay_api_key>data>value
     THREEPLAY_CALLBACK_KEY: __vault__:gen_if_missing:secret-operations/global/{{ business_unit }}/ocw-studio/threeplay_callback_key>data>value
     USE_X_FORWARDED_PORT: True
+    VIDEO_S3_TRANSCODE_ENDPOINT: __vault__::secret-operations/global/{{ business_unit }}/ocw-studio/{{ environment }}/video_s3_transcode_endpoint>data>value
```

Comment:
Because this is specific to OCW Studio I would just change the path to `__vault__::secret-ocw-studio/video_s3_transcode_endpoint`. There isn't a need to embed the environment because of the fact that we now have separate Vault clusters for each of our environment stages (CI/QA/Production)